lib/internal/streams/writable.js

```diff
@@ -323,7 +319,7 @@ function WritableState(options, stream, isDuplex) {
   // Crypto is kind of old and crusty.  Historically, its default string
   // encoding is 'binary' so we have to make this configurable.
   // Everything else in the universe uses 'utf8', though.
-  const defaultEncoding = options?.defaultEncoding;
+  const defaultEncoding = options ? options.defaultEncoding : null;
```

Comment:
It's a bit logic different here, compared to the previous logic, if the intention was to use `null` I suggest to use:
```suggestion
  const defaultEncoding = options?.defaultEncoding || null;
```
because by using the ternary operator here, we can get undefined, when the `options` have no defaultEncoding property:
```js
options = {}
const defaultEncoding = options ? options.defaultEncoding : null;
// defaultEncoding => undefined
```
it seems like it does matter due to the next line, that pay attention to `null`:

https://github.com/nxtedition/node/blob/ab94e41a49cd234848a2b75b479a6083b8bc6f02/lib/internal/streams/writable.js#L323

Reply:
yeah, it is, however, as I said if the intention was to use `null` when `defaultEncoding` is not specified, it's better to use `||`.